launching-workflows: mention launcher badge generator (#187)
reanahub/docs.reana.io: docs/running-workflows/launching-workflows/index.md

![Launch with Snakemake REANA at CERN](https://www.reana.io/static/img/badges/launch-with-snakemake-on-reana-at-cern.svg)

> If you would like to generate your own custom badge, you can use external services such as [shields.io](https://shields.io) or [badgen.net](https://badgen.net).

Reviewer: I think we should keep the part about generating custom badge designs using external services, as the badge creator is not a substitute for it (e.g. there is no way to change the text inside the badge, etc.)

> To streamline the badge creation process, you can use the tool accessible on REANA at the

Reviewer: What if we keep the sections about Markdown and HTML, and add a new one before those for the badge creator? This way we can show both the "automatic" way to generate badges, and the "manual" way to do it with Markdown and HTML.

What do you think?

> To streamline the badge creation process, you can use the tool accessible on REANA at the

Reviewer: Maybe let's explain that this is instance-dependent?
Something like: ... accessible on your REANA instance at `/launcher-badge`. If you have a CERN account, you can make use of launch badge creator of the available REANA@CERN instance.

Author: Okay, I agree! Should we do something similar for the other examples in the rest of the page? Even though maybe for them it's probably better to leave `reana.cern.ch`...

[`/launcher-badge`](https://reana.cern.ch/launcher-badge) URL.

Upon visiting the launcher badge page, you will be able to generate the necessary markdown syntax to include the badge

> in your documentation. This tool also provides a generic URL for initiating the workflow, which can be integrated into

Reviewer: Something like:

```suggestion
in your documentation. This tool also provides you the URL for executing your workflow, which can be integrated into
```

To emphasize that the tool generates the URL specific to a workflow, and not a generic one that needs to be customised. What do you think?
